app/src/components/misc/ProductWalkthrough/types.ts

```diff
@@ -0,0 +1,15 @@
+import { Step } from "react-joyride";
+import { TooltipRenderProps } from "react-joyride";
+
+export type pointerPlacement = "left" | "center" | "right" | "left-half" | "right-half";
+export interface CustomSteps extends Step {
+  disableNext?: (ruleData: any) => boolean;
+  showNext?: boolean;
+  autoMoveToNext?: boolean;
+  pointerPlacement?: pointerPlacement;
+}
+
+export interface CustomTooltipProps extends Omit<TooltipRenderProps, "step"> {
+  step: CustomSteps;
+  context: unknown;
+}
```

Comment:
![image](https://github.com/requestly/requestly/assets/16779465/417fd350-a953-47d6-9dd4-237301d25ae5)
6th step in the response rule should be to select static or programmatic mode. Else i can add anything in the programmatic mode in tour (Since i cant select programatic mode)